website/docs/docs/cloud/about-cloud/regions-ip-addresses.md

```diff
@@ -14,6 +14,7 @@ dbt Cloud is [hosted](/docs/cloud/about-cloud/architecture) in multiple regions
 |--------|----------|------------|--------------|----------------|-----------|-----------------|
 | North America [^1] | AWS us-east-1 (N. Virginia) | **Multi-tenant:** cloud.getdbt.com <br /> **Cell based:** ACCOUNT_PREFIX.us1.dbt.com | 52.45.144.63 <br /> 54.81.134.249 <br />52.22.161.231 <br />52.3.77.232 <br />3.214.191.130 <br />34.233.79.135 | ✅ | ✅ | ✅ |
 | EMEA [^1] | AWS eu-central-1	(Frankfurt) | emea.dbt.com | 3.123.45.39 <br /> 3.126.140.248 <br /> 3.72.153.148 | ❌ | ❌ | ✅ |
+| EMEA [^1] | Azure <br /> (Ireland) <Lifecycle status='beta' /> |     Beta invites only     | 20.13.190.192/26                                     | ❌ | ❌ | ✅ |
```

Comment:
The `Access URL` column for this should be 

**Cell based:** ACCOUNT_PREFIX.eu2.dbt.com